lib/kitchen/directions/bake_chapter_introductions/bake_chapter_objectives.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeChapterIntroductions
+  class BakeChapterObjectives
+    def bake(chapter:, chapter_objectives_strategy:)
+      chapter_outline_html = ''
```

Comment:
What do you think to rename `chapter_outline_html` to `chapter_objectives_html` and `chapter_objectives` to `chapter_objectives_note` or something like that?

![image (8)](https://user-images.githubusercontent.com/40228854/130611898-cb099e83-d6ee-462c-a7be-62e050d34938.png)

Later in `BakeChapterOutline` we are using this name, so maybe we can keep it consisntent.

![image (9)](https://user-images.githubusercontent.com/40228854/130612222-96c9b9a5-10ef-43ae-8529-86093f5a7886.png)

It's just a suggestion :wink: Let me know your thoughts.